src/azul/openapi/spec.py

```diff
@@ -232,3 +232,15 @@ def version(self) -> JSON:
                 }
             }
         }
+
+    @classmethod
```

Comment:
The fact that this is a class method while the other methods are not, is a smell. The method serves the same purpose as the other methods but the usage pattern is different, and that is the smell.